surfinBH/_fit_evaluators/fit_7dq4Emri.py

```diff
@@ -0,0 +1,224 @@
+
+    import surfinBH
+
+    # Load the fit
+    fit = surfinBH.LoadFits('NRSur7dq4EmriRemnant')
+
+    We provide the following call methods:
+        # remnant mass and 1-sigma error estimate
+        mf, mf_err = fit.mf(q, chiA, chiB, **kwargs)
+
+        # remnant spin and 1-sigma error estimate
+        chif, chif_err = fit.chif(q, chiA, chiB, **kwargs)
+        
+        # All of these together (this fit returns vf = vf_err = None)
+        # This shape of output is needed for compatibility reason
+        mf, chif, vf, mf_err, chif_err, vf_err = 
+            fit.all(q, chiA, chiB, **kwargs)
+
+    The arguments for each of these call methods are as follows:
+    Arguments:
+        q:     Mass ratio (q = mA/mB >=1)
+
+        chiA:  Dimensionless spin vector of the heavier black hole at
+                reference epoch.
+
+        chiB:  Dimensionless spin vector of the lighter black hole at
+                reference epoch.
+
+                This follows the same convention as LAL, where the spin
+                components are defined as:
+                \chi_z = \chi \cdot \hat{L}, where L is the orbital angular
+                    momentum vector at the epoch.
+                \chi_x = \chi \cdot \hat{n}, where n = body2 -> body1 is the
+                    separation vector at the epoch. body1 is the heavier body.
+                \chi_y = \chi \cdot \hat{L \cross n}.
+                These spin components are frame-independent as they are defined
+                using vector inner products. This is equivalent to specifying
+                the spins in the coorbital frame at the reference epoch. 
+                The reference epoch is assumed to be at t=-100 M from the peak 
+                of the waveform, see THE PAPER for additional details.
+
+    Optional arguments:
+        allow_extrap:
+            If False, raises a warning when q > 1000.1 or |chiA|,|chiB| > 0.81,
+                and raises an error when |chiA|,|chiB| > 1.
+            If True, allows extrapolation to any q and |chiA|,|chiB| <= 1.
+                Use at your own risk.
+            Default: False.
+
+    Inertial frame for returned values:
+
+        The returned chif are in the LAL inertial frame defined as follows:
+            The +ve z-axis is along the orbital angular momentum at the
+            reference epoch. The separation vector from the lighter BH to the
+            heavier BH at the reference epoch is along the +ve x-axis. The
+            y-axis completes the right-handed triad.
+
+            Note that the default reference epoch corresponds to t=-100M.
+            See LIGO DCC document T1800226 for the LAL frame diagram.
+    """
+
+    #-------------------------------------------------------------------------
+    def __init__(self, name):
+
+        # Param limits beyond which to raise a warning
+        soft_param_lims = {
+            'q': 1000.1,
+            'chiAmag': 0.81,
+            'chiBmag': 0.81,
+                }
```

Comment:
Set these to np.infty, 1, 1.
You have training data at chi=1 in the EMRI part, so this is not unreasonable.